src/app/contextBroker/contextBroker.cpp

```diff
@@ -371,6 +372,9 @@ PaArgument paArgs[] =
 #define IENTATTRVAL_COMPS_V2    6, { "v2", "entities", "*", "attrs", "*", "value" }
 #define IENTATTRVAL_COMPS_WORD  ""
 
+#define ETT                     EntityAllTypesRequest
+#define ETT_COMPS_V2            2, { "v2", "type" }
```

Comment:
type -> types

(We shoul also check the route for /v2/types/<type>, although it was developed in a previous PR).